python/observability-online-eval/agno/agent.py

```
@@ -0,0 +1,84 @@
+    tools=[YFinanceTools(stock_price=True, analyst_recommendations=True, company_info=True)],
+    instructions="Use tables to display data",
+    markdown=True,
+)
+
+# Aggregate both agents into a multi-agent system
+multi_ai_agent = Agent(
+    team=[web_search_agent, finance_agent],
+    model=Gemini(id="gemini-2.0-flash-001"),
+    instructions="You are a helpful financial assistant. Answer user questions about stocks, companies, and financial data.",
+    show_tool_calls=True,
+    markdown=True
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)